### DISCUSSION TITLE
Log search terms

### DISCUSSION THREAD

**Discussion by ngosang** [benbusby/whoogle-search#599] [Q&A]
Is it possible to log search terms?
If not, is it possible to have some kind of access log?

**Comment by Albonycal** [comment]
This project does not promote logging (tracking etc.) 
but ig you can use a proxy to easily keep a log of things

**Reply by ngosang** [reply]
I'm using access logs from Traefik (with GET query) for now.

IMHO this feature should be implemented in Whoogle for several reasons:
* Debug bad configuration / query errors
* Statistics about users, search terms, HTTP response codes...
* Measure query time
